bootstrap/aws/config-default.sh

```diff
@@ -13,7 +13,8 @@ ZONE=${APOLLO_AWS_ZONE:-eu-west-1b}
 MASTER_SIZE=${MASTER_SIZE:-m1.medium}
 SLAVE_SIZE=${SLAVE_SIZE:-m1.medium}
 NUM_SLAVES=${NUM_SLAVES:-1}
-
+FRAMEWORK_MARATHON_ENABLED=${FRAMEWORK_MARATHON_ENABLED:-'Y'}
```

Comment:
seems to be alot of duplication in the aws and digitalocean bootstrap scripts, can we make a common bootstrap, so when we add more providers or frameworks later its easier